analyzer/config/docker/docker_test.go

```diff
@@ -0,0 +1,184 @@
+package docker
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/open-policy-agent/conftest/parser/docker"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/analyzer/config"
+	"github.com/aquasecurity/fanal/types"
+)
+
+func Test_dockerConfigAnalyzer_Analyze(t *testing.T) {
+	tests := []struct {
+		name      string
+		inputFile string
+		want      *analyzer.AnalysisResult
+		wantErr   string
+	}{
+		{
+			name:      "happy path",
+			inputFile: "testdata/Dockerfile.deployment",
+			want: &analyzer.AnalysisResult{
+				Configs: []types.Config{
+					{
+						Type:     config.Dockerfile,
+						FilePath: "testdata/Dockerfile.deployment",
+						Content: []interface{}{
+							[]interface{}{
+								map[string]interface{}{
+									"Cmd":    "from",
+									"SubCmd": "",
+									"JSON":   false,
+									"Flags":  []interface{}{},
+									"Value":  []interface{}{"foo"},
+								},
+								map[string]interface{}{
+									"Cmd":    "copy",
+									"SubCmd": "",
+									"JSON":   false,
+									"Flags":  []interface{}{},
+									"Value":  []interface{}{".", "/"},
+								},
+								map[string]interface{}{
+									"Cmd":    "run",
+									"SubCmd": "",
+									"JSON":   false,
+									"Flags":  []interface{}{},
+									"Value":  []interface{}{"echo hello"},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		{
+			name:      "happy path with multi-step",
```

Comment:
If I understand correctly, it is "multi-stage` rather than "multi-step", right?

```suggestion
			name:      "happy path with multi-stage",
```

Reply:
fixed.